### PR TITLE
Tree span distribution

### PR DESCRIPTION
**Pull request by savitakartik**

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Cool! Can you put some examples in here of what they look like with and without log-transform?

**Comment by savitakartik** [comment]
Here's an example without the log transform:
image
and one with it:
image

**Comment by jeromekelleher** [comment]
Nice :+1:

**Comment by jeromekelleher** [comment]
Needs a rebase now. The trick here will be to drop the first commit, because that conflicts with the PR we just merged. You drop a commit by deleting that line in the rebase "file" that you get to edit.

File: utils.py [hunk 1]
```diff
@@ -220,6 +220,52 @@ def plot_mutations_per_node(self, max_num_muts=None, show_counts=False):
         )
         plt.xlabel("Number of mutations")
         plt.ylabel("Number of nodes")
+<<<<<<< HEAD
```
Comment (jeromekelleher) [inline utils.py:223]: Whoops, some conflicts to resolve here!

Comment (savitakartik) [reply utils.py]: Ah, thanks. I think I've resolved all conflicts now?!

File: utils.py [hunk 2]
```diff
@@ -223,3 +223,44 @@ def plot_mutations_per_node(self, max_num_muts=None, show_counts=False):
...
+                spans, alpha=0.75, edgecolor="black")
+            ax.xaxis.set_major_locator(plt.MaxNLocator(10))
+        if show_counts:
+            plt.bar_label(bars, fmt="{:,.0f}")
+        ax.set_xlabel(xlabel)
+        ax.yaxis.set_major_formatter(
+            plt.FuncFormatter(lambda x, pos: "{:,}".format(int(x)))
+        )
+        plt.title(f"Distribution of {len(spans):,} tree spans")
+>>>>>>> 1d332fd (Fixed issues with hist: (a)to include final log bin which was being omitted in the plot (b)to remove non-informative first span.)
```
Comment (jeromekelleher) [inline utils.py:266]: Also here

**Comment by jeromekelleher** [comment]
Now that you've fixed the conflicts it would be good to squash down to one commit. (The stdpopsim guide talks about this)

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**